docs/architecture/adr/0027-angular-i18n-migration.md

```diff
@@ -0,0 +1,185 @@
+---
+adr: "0027"
+status: Proposed
+date: 2025-08-01
+tags: [clients, angular, i18n]
+---
+
+# 0027 - Improved Localization
+
+<AdrTable frontMatter={frontMatter}></AdrTable>
+
+## Context and Problem Statement
+
+We currently use a homegrown localization system based on the
+https://developer.chrome.com/docs/extensions/reference/api/i18n api. This has served us reasonably
```

Comment:
🎨 Make into a prettier link and not just show the URL? There are a few of these on this page.